src/Sharpliner/AzureDevOps/StaticVariablesReference.cs

```diff
@@ -4,5 +4,5 @@ namespace Sharpliner.AzureDevOps;
 
 public class StaticVariablesReference : VariablesReference
```

Comment:
Don't think you need this now either

Reply:
Yup, removed

Reply:
Here: https://github.com/sharpliner/sharpliner/pull/238/commits/5207e04e6401cb614820ee44937d0ad24e212a9c